packages/venia-concept/jest.config.js

```diff
@@ -4,5 +4,8 @@ module.exports = {
     moduleNameMapper: {
         '\\.css$': 'identity-obj-proxy',
         '^src/(.+)': '<rootDir>/src/$1'
-    }
+    },
+    // We don't ship Peregrine with CSJ support (intentional), so transpile
+    // used Peregrine code in tests
+    transformIgnorePatterns: ['node_modules/(?!@magento/peregrine)']
```

Comment:
Letting `jest` take care of transpiling `peregrine` code used in `venia` during tests, because we don't want to a second persistent process running to build artifacts to disk just for tests